### DISCUSSION TITLE
Consider adding a codespaces setup

### DISCUSSION THREAD

**Discussion by markgoho** [openai/whisper#660] [Ideas]
In addition to providing a long list of commands to make one's own local environment ready to use the Whisper model, consider adding a devcontainer.json (Codespaces) setup with all of this stuff already available so that users that want to try it out have to just click a single button that will spin up a cloud-based environment with all necessary software already installed.

**Comment by okanji** [comment]
That would be immensely helpful. And if anyone has their own devcontainer.json file for whisper that they can share, that would be much appreciated.
